### PR TITLE
Bugfix scripts and links security vulnerabilities

### PR DESCRIPTION
**Pull request by kbeker**
This is part of #507 issue. First part with django settings is already merged into master. This pull request is especially to test all views by @Karrp because all script and stylesheets have changed tags.

### CODE REVIEW

File: employees/templates/employees/author_report_list.html [hunk 1]
```diff
@@ -7,8 +7,19 @@
...
+    rel="stylesheet"
+    href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css"
+    integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+    crossorigin="anonymous"
+/>
+<link
+    rel="stylesheet"
+    type="text/css"
+    href="{% static 'employees/style.css' %}"
+    integrity="{% staticinline "employees/style.css" encode="sri" %}"
```
Comment (Szymiks) [inline employees/templates/employees/author_report_list.html]: You are not loading `staticinline`, here.

File: employees/templates/employees/project_report_detail.html [hunk 2]
```diff
@@ -5,7 +5,13 @@
 {% load crispy_forms_tags %}
 
 {% block extra_head %}
-<link rel="stylesheet" type="text/css" href="{% static 'employees/style.css' %}">
+<link
+    rel="stylesheet"
+    type="text/css"
+    href="{% static 'employees/style.css' %}"
+    integrity="{% staticinline "employees/style.css" encode="sri" %}"
```
Comment (Szymiks) [inline employees/templates/employees/project_report_detail.html]: Same here

File: employees/templates/employees/project_report_list.html [hunk 3]
```diff
@@ -7,8 +7,19 @@
...
+    rel="stylesheet"
+    href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css"
+    integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+    crossorigin="anonymous"
+/>
+<link
+    rel="stylesheet"
+    type="text/css"
+    href="{% static 'employees/style.css' %}"
+    integrity="{% staticinline "employees/style.css" encode="sri" %}"
```
Comment (Szymiks) [inline employees/templates/employees/project_report_list.html]: Same here

File: employees/templates/employees/report_detail.html [hunk 4]
```diff
@@ -5,8 +5,19 @@
...
+    rel="stylesheet"
+    href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css"
+    integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+    crossorigin="anonymous"
+/>
+<link
+    rel="stylesheet"
+    type="text/css"
+    href="{% static 'employees/style.css' %}"
+    integrity="{% staticinline "employees/style.css" encode="sri" %}"
```
Comment (Szymiks) [inline employees/templates/employees/report_detail.html]: Same here

File: employees/templates/employees/report_detail.html [hunk 5]
```diff
@@ -43,13 +43,22 @@ <h1>{{ UI_text.PAGE_TITLE.value }}{{ report.project }} ({{ report.date }})</h1>
...
+        integrity="sha384-JPbtLYL10d/Z1crlc6GGGGM3PavCzzoUJ1UxH0bXHOfguWHQ6XAWrIzW+MBGGXe5"
+        crossorigin="anonymous"></script>
+    <script type="text/javascript">
+        var discard_text = "{{ UI_text.DELETE_POPUP_NO.value }}";
+        var confirmation_text = "{{ UI_text.DELETE_POPUP_YES.value }}";
+        var redirect_url =﻿"{% url 'custom-report-delete' pk=report.pk %}";
+    </script>
+    <script
+        src="{% static 'employees/scripts/basic_popup_window.js' %}"
+        integrity="{% staticinline 'employees/scripts/basic_popup_window.js' encode="sri" %}"
```
Comment (Szymiks) [inline employees/templates/employees/report_detail.html]: as before

**Comment by kbeker** [comment]
@Szymiks I deleted all `staticinline` loading becuase it doesn't affect on application. If anything will change in future I will fix it. Merging

**Pull request merged**